app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/RadioInput/__tests__/Radioinput.test.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
test files are not needed for common or reusable components as of now, plz remove them
plz don't just delete them,
better take a backup of these files
so if client asks we have then ready to put back

explanation: when we write proper test cases on the actual component i.e. form or table, we will have covered all the common component functionality and behavior
so we do not have to write test cases for simple JSX components that do not have any functionality or state management
this may not be true always, so please understand the concept carefully and then learn and implement accordingly

I need proper response if this is understood or there are any gaps

Reply:
This Folder is removed and it's code is shifted to InputWidget